utils/build-windows-toolchain.bat

```diff
@@ -77,6 +77,7 @@ call :CloneRepositories || (exit /b 1)
 
 :: We only have write access to BuildRoot, so use that as the image root.
 powershell.exe -ExecutionPolicy RemoteSigned -File %~dp0build.ps1 ^
+  -HostArchName %HostArchName% ^
```

Comment:
Is this guaranteed to be set always? This is not an optional parameter, it is just a default valued parameter.

Reply:
Yeah, you basically want to match `WindowsSDKArchitecturesArg`